frontend/javascripts/oxalis/model_initialization.js

```diff
@@ -452,21 +421,53 @@ function setupLayerForVolumeTracing(
   const fallbackLayer = layers[fallbackLayerIndex];
   const boundaries = getBoundaries(dataset);
 
+  console.log(tracing.resolutions);
```

Comment:
If this should be kept, I would add a label to it in the output, otherwise I would remove it.